playbooks/kubevirt.yml

```diff
@@ -0,0 +1,253 @@
+---
+- hosts: all
+  become: true
+  vars:
+    selinux_state: permissive
+    pod_network: 10.244.0.0/16
+    flannel_version: a70459be0084506e4ec919aa1c114638878db11b
+    multus_version: master
+    kubevirt_version: v0.13.2
+    ovs_cni_version: master
+  roles:
+    - selinux
+    - epel_repositories
+  tasks:
+    - name: disable swap
+      command: swapoff -a
+
+    - name: deconfigure swap
+      mount:
+        src: /swapfile
+        fstype: swap
+        path: none
+        state: absent
+    - name: modprobe br_netfilter
+      modprobe:
+        name: br_netfilter
+
+    - name: configure bridge iptables
+      sysctl:
+        name: "{{ item }}"
+        value: 1
+        sysctl_file: /etc/sysctl.d/k8s.conf
+      with_items:
+        - net.bridge.bridge-nf-call-ip6tables
+        - net.bridge.bridge-nf-call-iptables
+
+    - name: install docker
+      yum:
+        name: docker
+
+    - name: enable docker
+      service:
+        name: docker
+        enabled: true
+        state: started
+
+    - name: enable CentOS OpenStack repository
+      yum_repository:
+        name: centos-openstack-rocky
+        description: CentOS OpenStack
+        baseurl: http://mirror.centos.org/centos/$releasever/cloud/$basearch/openstack-rocky/
+        enabled: true
+        gpgcheck: false
+
+    - name: install openvswitch
+      yum:
+        name: openvswitch
+
+    - name: enable openvswitch
+      service:
+        name: openvswitch
+        enabled: true
+        state: started
+
+    - name: add foreman bridge
+      openvswitch_bridge:
+        bridge: foreman
+        state: present
+
+    - name: add port to foreman bridge
+      openvswitch_port:
+        bridge: foreman
+        port: eth1
+        state: present
+
+
+    - name: k8s repo
+      yum_repository:
+        name: kubernetes
+        description: Kubernetes
+        baseurl: https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64
+        enabled: true
+        gpgcheck: true
+        repo_gpgcheck: true
+        gpgkey: https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+        exclude: kube*
+
+    - name: install kubelet kubeadm kubectl
+      yum:
+        name:
+          - kubelet
+          - kubeadm
+          - kubectl
+        disable_excludes: kubernetes
+
+    - name: enable kubelet accounting
+      copy:
+        dest: /etc/systemd/system/kubelet.service.d/11-cgroups.conf
```

Comment:
Dest directory is not correct. Based on kubelet rpm spec the correct directory is: `/usr/lib/systemd/system/kubelet.service.d`.

Reply:
`/usr/lib` is for packages, we're overriding packaged defaults, so that goes into `/etc` (this is also what's documented in https://github.com/masayag/foreman_kubevirt/blob/master/getting-started.md)

Reply:
OK so either we override it or change to `/usr/lib`. Currently it is failing.